### PR TITLE
bdex/80773: Toxic Exposure - [Lighthouse API] generatePdf hook up

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- *This work is behind a feature toggle (flipper): YES*
-  enable benefits_claims/service#submit526 to call LH's /generatePDF endpoint

## Related issue(s)

- #80773

## Testing done
Yes
- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: lib/lighthouse/benefits_claims/configuration.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ module BenefitsClaims
   class Configuration < Common::Client::Configuration::REST
     self.read_timeout = Settings.lighthouse.benefits_claims.timeout || 20
 
-    API_SCOPES = %w[system/claim.read system/claim.write].freeze
+    API_SCOPES = %w[system/claim.read system/claim.write system/526-pdf.override].freeze
```
Comment (aurora-a-k-a-lightning) [inline lib/lighthouse/benefits_claims/configuration.rb:16]: verified that staging and production Lighthouse benefits claims client ids can generate an auth token with these scopes listed
references: https://dsva.slack.com/archives/C02CQP3RFFX/p1713469011121339

**Review by sethdarragile6 — APPROVED** [review]
Looks great! Approved!

**Comment by rmtolmach** [comment]
@aurora-a-k-a-lightning are there any tests in `spec/lib/lighthouse/benefits_claims/service_spec.rb` that can be updated to go along with your code changes?

**Review by sethdarragile6 — APPROVED** [review]

**Comment by aurora-a-k-a-lightning** [comment]
@rmtolmach ah yes :)
there is a test now.
thank you!

**Review by rmtolmach — APPROVED** [review]
LGTM!

**Pull request merged**